### DISCUSSION TITLE
can't run mojo file

### DISCUSSION THREAD

**Discussion by BlackHo0d** [mojolicious/mojo#2119] [Q&A] [answered]
I've installed mojo and now can use it in the REPL.  But I can't run mojo files. I've read that I should add the following lines to your configuration file (.bash_profile, .bashrc, or .zshrc) but I don't understand how to do it. Can anyone help me with that ? 

I'm using Mac OS(Intel)

**Comment by kraih** [comment]
Are you sure this is the right project?

**Reply by BlackHo0d** [reply]
I've created the file hello.mojo and attempted to run but I can't and don't understand what I should fix

**Reply by kraih** [reply]
Yes, this is not the open source project you are looking for.

**Reply by BlackHo0d** [reply]
I don't understand you. What should I do ?

**Reply by Akron** [reply, marked as answer]
Maybe you are looking for help with [this mojo](https://github.com/modularml/mojo/discussions)?

**Reply by BlackHo0d** [reply]
Yeah, thank you
